documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -54,6 +54,13 @@ For example `ca-2018-09-27T17-32-00Z.crt`.
 +
 [source,shell,subs="+quotes"]
 mv ca.crt ca-$(date -u -d"$(openssl x509 -enddate -noout -in ca.crt | sed 's/.*=//')" +'%Y-%m-%dT%H-%M-%SZ').crt
++
+If you are using a different key to generate a new certificate, retain the current CA certificate.
+The CN (Common Name) of the new CA certificate must also be different from the current one.
+For example, when the Cluster Operator renews certificates automatically it adds a _v<version_number>_ suffix to identify a version.
+Do the same with your own certificates.
+If you are using the same key for generate a new CA certificate, don't retain the old one in the secret.
```

Comment:
I think this deserves some more explanation => might be useful to explain why but also what does retaining men.